data/json/items/gun/36paper.json

```diff
@@ -1,29 +1,31 @@
+Cataclysm web linting tool
```

Comment:
Whoops thought i fixed that.... This happend when my VM was dying and copy and paste were getting weird